app/controllers/Pref.scala

```diff
@@ -98,11 +98,17 @@ final class Pref(env: Env) extends LilaController(env):
     "bg"           -> (forms.bg           -> save("bg")),
     "bgImg"        -> (forms.bgImg        -> save("bgImg")),
     "is3d"         -> (forms.is3d         -> save("is3d")),
-    "zen"          -> (forms.zen          -> save("zen")),
+    "zen"          -> (forms.zen          -> saveZen()),
     "voice"        -> (forms.voice        -> save("voice")),
     "keyboardMove" -> (forms.keyboardMove -> save("keyboardMove"))
   )
 
+  private def saveZen()(value: String, ctx: Context): Fu[Cookie] =
+    //Discard 0-1 bit, then add 0-1 bit with wanted value
+    var newVal = ctx.pref.zenInt & ~1 | value.toInt
```

Comment:
this is quite cryptic, even with the comment. Why?

Reply:
Zen.GAME_AUTO (=2) was initially Zen.Automatic_Bit [with Zen.Game_On ('3') and Zen.Game_Off, with same '2' value] when I started developping it ; this helped understand that it is a bit functionality (I had 3 var, but 2 were unused so no point in keeping them in the end). 

This makes it so the user can keep toggling it's zen, and that whatever he does, when he toggles it, it will only affect the "yes-no" part of it, and never the "Automatic-in-game" part of it (while can only be activated/deactivated via the pref menu).

This reduces the knowledge one needs regarding toggling zen, and allows to not have on every zenable page  conditionnal setting of value (e.g. if zen is automatically-in-game or not).

Reply:
I see but the problem with this is that it leaks the zen storage binary subtleties to the Pref controller.

Reply:
Would moving saveZen in a helper file solve this ? Or should I put more conditional logic in the .ts ?